fiddler/datadog_checks/fiddler/config_models/__init__.py

```diff
@@ -0,0 +1,21 @@
+# This file is autogenerated.
+# To change this file you should edit assets/configuration/spec.yaml and then run the following commands:
+#     ddev -x validate config -s <INTEGRATION_NAME>
+#     ddev -x validate models -s <INTEGRATION_NAME>
```

Comment:
Can you please run `ddev -x validate models -s fiddler` so that the config models files can be updated to reflect the values from the `spec.yaml` file?

Reply:
done.